internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go

```diff
@@ -111,22 +110,84 @@ func importStateIDFunc(resourceName string) resource.ImportStateIdFunc {
 	}
 }
 
-func configBasic(projectID, bucketName, iamRoleID string) string {
+func configBasic(projectID, bucketName string) string {
+	bucketResourceName := "arn:aws:s3:::" + bucketName
 	return fmt.Sprintf(`
-		resource "mongodbatlas_cloud_backup_snapshot_export_bucket" "test" {
-			project_id     = "%[1]s"
-    	  	iam_role_id    = "%[3]s"
-       		bucket_name    = "%[2]s"
-       		cloud_provider = "AWS"
-    	}
-
-		data "mongodbatlas_cloud_backup_snapshot_export_bucket" "test" {
-			project_id   =  mongodbatlas_cloud_backup_snapshot_export_bucket.test.project_id
-			export_bucket_id = mongodbatlas_cloud_backup_snapshot_export_bucket.test.export_bucket_id
-		}
+    resource "aws_iam_role_policy" "test_policy" {
+        name = "mongodb-atlas-policy-export-bucket"
```

Comment:
better use acc.RandomIAMRole instead of fixed name

Reply:
+1